Automattic/wp-calypso
# Update Plans page tooltip copy

This updates the Plans page tooltip copy as discussed in #22701 — specifically, @benhuberman's comments [here](https://github.com/Automattic/wp-calypso/issues/22701#issuecomment-369704447).

**Testing instructions:**

- checkout this branch
- visit http://calypso.localhost:3000/plans/YOURTESTINGSITEURL and confirm the changes align with Ben's suggestions.
